data/com.endlessm.Clippy.Extension.appdata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>com.endlessm.Clippy.Extension</id>
+  <extends>com.endlessm.HackComponents</extends>
+  <metadata_license>LicenseRef-proprietary</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>Clippy</name>
+  <summary>Clips things together</summary>
```

Comment:
Meh

How about "Component that makes apps hackable"?

Reply:
This is what is in the Clippy README. I found it amusing.